cloud-azure/src/main/java/com/sequenceiq/cloudbreak/cloud/azure/AzureTerminationHelperService.java

```diff
@@ -71,78 +77,97 @@
     public List<CloudResourceStatus> downscale(AuthenticatedContext ac, CloudStack stack, List<CloudInstance> vms,
             List<CloudResource> allResources, List<CloudResource> resourcesToRemove) {
         List<CloudResource> networkResources = azureCloudResourceService.getNetworkResources(allResources);
-        return terminateResources(ac, stack, vms, resourcesToRemove, networkResources, false);
+        return terminateResources(ac, stack, vms, resourcesToRemove, networkResources, TerminationOperations.DOWNSCALE);
     }
 
     public List<CloudResourceStatus> terminate(AuthenticatedContext ac, CloudStack stack, List<CloudResource> resourcesToRemove) {
         List<CloudInstance> vms = new ArrayList<>();
         stack.getGroups().forEach(group -> vms.addAll(group.getInstances()));
         List<CloudResource> networkResources = azureCloudResourceService.getNetworkResources(resourcesToRemove);
-        return terminateResources(ac, stack, vms, resourcesToRemove, networkResources, true);
+        return terminateResources(ac, stack, vms, resourcesToRemove, networkResources, TerminationOperations.TERMINATE);
     }
 
     private List<CloudResourceStatus> terminateResources(AuthenticatedContext ac, CloudStack stack, List<CloudInstance> vms,
-            List<CloudResource> resourcesToRemove, List<CloudResource> networkResources, boolean deleteWholeDeployment) {
+            List<CloudResource> resourcesToRemove, List<CloudResource> networkResources, TerminationOperations operation) {
         LOGGER.debug("Terminating the following resources: {}", resourcesToRemove);
-        LOGGER.debug("Operation is: {}", deleteWholeDeployment ? "terminate" : "downscale");
+        LOGGER.debug("Operation is: {}", operation.toString().toLowerCase(Locale.ROOT));
```

Comment:
don't over complicate please, just use `operation`
also these 2 logs could be joined into one. I don't see any reason for 2 statements

Reply:
I've changed this to use `operation`.

I'd argue the two log statements is more readable than:
```java
        LOGGER.debug("Terminating the following resources: {}\nOperation is: {}", resourcesToRemove, operation);
```

Reply:
I'm going to side with Mihaly here. You could massage the language a little more.
```
LOGGER.debug("Initiating operation {} to terminate the following resources: {}", operation, resourcesToRemove);
```